docs/4.2/ibm_cloud_guide.md

```diff
@@ -0,0 +1,179 @@
+
+When setting up `audit_session_uri` use `s3://` session prefix.
+
+The credentials are used from `~/.aws/credentials` and should be created with HMAC option:
+
+![Creating Service Creds](img/IBM/cloud.ibm.com_object-store-service-creds.png) 
+
+```json 
+{
+  "apikey": "LU9VCDf4dDzj1wjt0Q-BHaa2VEM7I53_3lPff50d_uv3",
+  "cos_hmac_keys": {
+    "access_key_id": "e668d66374e141668ef0089f43bc879e",
+    "secret_access_key": "d8762b57f61d5dd524ccd49c7d44861ceab098d217d05836"
+  },
+  "endpoints": "https://control.cloud-object-storage.cloud.ibm.com/v2/endpoints",
+  "iam_apikey_description": "Auto-generated for key e668d663-74e1-4166-8ef0-089f43bc879e",
+  "iam_apikey_name": "Service credentials-1",
+  "iam_role_crn": "crn:v1:bluemix:public:iam::::serviceRole:Writer",
+  "iam_serviceid_crn": "crn:v1:bluemix:public:iam-identity::a/0328d127d04047548c9d4bedcd24b85e::serviceid:ServiceId-c7ee0ee9-ea74-467f-a49e-ef60f6b27a71",
+  "resource_instance_id": "crn:v1:bluemix:public:cloud-object-storage:global:a/0328d127d04047548c9d4bedcd24b85e:32049c3c-207e-4731-8b8a-53bf3b4844e7::"
+}
+```
+
+Save these settings to `~/.aws/credentials` 
+
+```yaml
+# Example keys from example service account to be saved into ~/.aws/credentials
+[default]                              
+access_key_id="e668d66374e141668e3432443bc879e"
+secret_access_key="d8762b57f61d5dd524ccd49c7d44861ceafdsfds37d05836"
+```
+
+Example `/etc/teleport.yaml`
+```yaml
+...
+storage:
+    ...
+    # Note
+    # 
+    # endpoint=s3.us-east.cloud-object-storage.appdomain.cloud | This URL will 
+    # differ depending on which region the bucket is created. Use the  public 
+    # endpoints.
+    # 
+    # region=ibm | Should always be set as IBM. 
+    audit_sessions_uri: 's3://BUCKETNAME/readonly/records?endpoint=s3.us-east.cloud-object-storage.appdomain.cloud&region=ibm'
+    ...
+```
+
+!!!tip
+    When starting with `teleport start --config=/etc/teleport.yaml -d` you can confirm that the 
+    bucket has been created. 
+
+
+```bash
+root@teleport:~# teleport start --config=/etc/teleport.yaml -d
+DEBU [SQLITE]    Connected to: file:/var/lib/teleport/proc/sqlite.db?_busy_timeout=10000&_sync=OFF, poll stream period: 1s lite/lite.go:173
+DEBU [SQLITE]    Synchronous: 0, busy timeout: 10000 lite/lite.go:220
+DEBU [KEYGEN]    SSH cert authority is going to pre-compute 25 keys. native/native.go:104
+DEBU [PROC:1]    Using etcd backend. service/service.go:2309
+INFO [S3]        Setting up bucket "ben-teleport-test-cos-standard-b98", sessions path "/readonly/records" in region "ibm". s3sessions/s3handler.go:143
+INFO [S3]        Setup bucket "ben-teleport-test-cos-standard-b98" completed. duration:356.958618ms s3sessions/s3handler.go:147
+```
+
+### Network: IBM Cloud DNS Services
+
+We recommend using [IBM Cloud DNS](https://cloud.ibm.com/catalog/services/dns-services) for the Teleport Proxy public address.  
```

Comment:
Any more detail on this and what it should point to? Associated config for `public_addr` in the Teleport config file?